src/lucky/redirectable.cr

```diff
@@ -42,29 +42,29 @@ module Lucky::Redirectable
     redirect to: action.route, status: status
   end
 
+  # Redirect using a `String` and a `Status` value
+  #
+  # ```crystal
+  # redirect to: "/users", status: Status::MovedPermanently
+  # ```
+  # You can find a list of all of the possible statuses [here](https://github.com/luckyframework/lucky/blob/master/src/lucky/action.cr).
+  def redirect(to path : String, status : Lucky::Action::Status = Lucky::Action::Status::Found) : Lucky::TextResponse
+    redirect(path, status.value)
+  end
+
   # Redirect using a `String`
```

Comment:
While you're in here could you change this to `Redirect to the given path, with an optional `Int32` status`